transactions/update-contract/2023/feb-23/README.md

```diff
@@ -19,6 +19,88 @@ Used this to generate args:
 
 `cat "./FlowIDTableStaking.cdc" | xxd -p -c0 | tr -d '\n'`
 
+Verified using:
+```
+$ cat arguments-update-contract-FlowIDTableStaking-mainnet.json | jq '.[] | .value' | xxd -r -p > temp1.txt
+$ diff temp1.txt FlowIDTableStaking.cdc
+(Should produce no difference)
+```
+
+> Tools is unable to generate the link since the argument content is too large. We will have to do this using the old way using `flow cli`
+
+## Dapper Labs Builds
+
+```sh
+flow transactions build ./transactions/update-contract/2023/feb-23/upgrade-set-candidate.cdc \
+  --config-path flow-staking.json \
+  --network mainnet \
+  --args-json "$(cat "./transactions/update-contract/2023/feb-23/arguments-update-contract-FlowIDTableStaking-mainnet.json")" \
+  --proposer 0x8d0f8df62029f75a \
+  --proposer-key-index 0 \
+  --authorizer 0x8624b52f9ddcd04a \
+  --payer 0x8624b52f9ddcd04a \
+  -x payload \
+  --save ./transactions/update-contract/2023/feb-23/flow-id-table-staking-contract-upgrade-feb-23-unsigned.rlp
+```
+
+## Josh Signs
+
+```sh
+flow transactions sign ./transactions/update-contract/2023/feb-23/flow-id-table-staking-contract-upgrade-feb-23-unsigned.rlp \
+  --config-path flow-staking.json \
+  --signer vishal \
+  --filter payload \
+  --save ./transactions/update-contract/2023/feb-23/flow-id-table-staking-contract-upgrade-feb-23-sig-1.rlp
+```
+
+## Emerson Signs
+
+```sh
+flow transactions sign ./transactions/update-contract/2023/feb-23/flow-id-table-staking-contract-upgrade-feb-23-sig-1.rlp \
+  --config-path flow-staking.json \
+  --signer animoca \
+  --filter payload \
+  --save ./transactions/update-contract/2023/feb-23/flow-id-table-staking-contract-upgrade-feb-23-sig-2.rlp
+```
+
+## Alex Signs
+
+```sh
+flow transactions sign ./transactions/update-contract/2023/feb-23/flow-id-table-staking-contract-upgrade-feb-23-sig-2.rlp \
+  --config-path flow-staking.json \
+  --signer alexgrach \
+  --filter payload \
+  --save ./transactions/update-contract/2023/feb-23/flow-id-table-staking-contract-upgrade-feb-23-sig-3.rlp
+```
+
+## Kan Signs
+
+```sh
+flow transactions sign ./transactions/update-contract/2023/feb-23/flow-id-table-staking-contract-upgrade-feb-23-sig-3.rlp \
+  --config-path flow-staking.json \
+  --signer kan \
+  --filter payload \
+  --save ./transactions/update-contract/2023/feb-23/flow-id-table-staking-contract-upgrade-feb-23-sig-4.rlp
+```
+
+## Contract Admin Signs (do we need this?)
+
+```sh
```

Comment:
@Kay-Zee do we need this step? I saw this in an old update to the staking contract where there was this additional step of `contract admin signs` was done.

Reply:
no this is no longer needed

Reply:
I don't think we'll need this. The contract admin was something from the locked tokens contract I believe. The last upgrade we did did not require this